com.unity.render-pipelines.high-definition/Runtime/Material/Fullscreen/HDFullscreenFunctions.hlsl

```diff
@@ -0,0 +1,6 @@
+
+// For now we fake the GI sampling in fullscreen.
+float3 SampleBakedGI(float3 positionRWS, float3 normalWS, float2 uvStaticLightmap, float2 uvDynamicLightmap)
+{
+    return 0;
```

Comment:
I am not sure to understand why we need a new file just for this? why is it here?